### ISSUE TITLE
Evaluation code for MuPoTS-3D dataset

### ISSUE THREAD

**Issue by Kelly510** [ChiSu001/SAT-HMR#11]
Hi,

Thanks for the great work! I was wondering if you could share the evaluation code for the MuPoTS-3D dataset? I saw the results in the paper, but couldn’t find the code in the repo.

Appreciate your help!

**Comment by Kelly510** [comment]
By the way, are the hyperparameters used for fine-tuning on 3DPW the same as those used in the initial training stage? Many thanks!

**Comment by ChiSu001** [comment]
Hi @Kelly510 ,

The MuPoTS-3D evaluation uses their [official code](https://vcai.mpi-inf.mpg.de/projects/SingleShotMultiPerson/) which is written in matlab, and CMU Panoptic evaluation follows [ROMP](https://github.com/Arthur151/ROMP) and [BMP](https://github.com/jfzhang95/BMP).

For 3DPW fine-tuning, we use a smaller learning rate (1.0e-5).
